docs/api/topics/firefox_os_addons.rst

```diff
@@ -58,8 +58,8 @@ Add-on
     :type download_url: string
     :param name: The add-on name.
     :type name: string|object
-    :param manifest_url: The (absolute) URL to the mini-manifest for that add-on.
-    :type manifest_url: string
+    :param mini_manifest_url: The (absolute) URL to the mini-manifest for that add-on.
+    :type mini_manifest_url: string
```

Comment:
Our team is great at coming up with cutesy names that make sense to us, but this is outward-facing documentation to people who might be unfamiliar.